esi_leap/api/wsgi.py

```diff
@@ -0,0 +1,44 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import sys
+
+from oslo_log import log as logging
+
+from esi_leap.api import app
+from esi_leap.common import i18n
+from esi_leap.common import service
+import esi_leap.conf
+
+
+CONF = esi_leap.conf.CONF
+LOG = logging.getLogger(__name__)
+
+
+class EsiLeapApiWsgiApplication(object):
```

Comment:
I wouldn't hate it if this was 'ESILeapAPIWSGIApplication - although staring at that string kinda makes my eyes hurt so maybe not. I noticed the Ironic version has much simpler naming conventions, so maybe that's okay... ?

Reply:
It looks like ironic has a similar class in app.py. Do we want to move this class to app.py?

Reply:
hmm, i do think moving it to app.py would be a solid call and since it's already in the ESI-Leap repo and in the esi_leap.api namespace we could probably just call it "WSGIApplication" (or "WsgiApplication")